src/components/Proposal.jsx

```diff
@@ -138,6 +166,54 @@ const Proposal = ({ offer, purses }) => {
     </div>
   );
 
+  const executionFeeEntry = executionFee && (
+    <div className="OfferEntry">
+      <h6>
+        Pay Execution Fee
+        <>
+          <IconButton
+            color="font"
+            size="small"
+            aria-label="execution fee info"
+            onClick={handleClick}
+          >
+            <InfoOutlined fontSize="inherit" />
+          </IconButton>
+          <Popover
+            open={isOpen}
+            anchorEl={anchorEl}
+            onClose={handleClose}
+            anchorOrigin={{
+              vertical: 'bottom',
+              horizontal: 'left',
+            }}
+          >
+            <Box sx={{ p: 2, maxWidth: '480px' }}>
+              The on-chain computation cost of this transaction. These are
+              batched and deducted from your purse balance every 10
+              transactions.
```

Comment:
```suggestion
              batched and deducted from your purse balance once the total reaches 0.20 IST.
```

where 0.20 is computed from `minFeeDebit`